lib/host/wasi/wasifunc.cpp

```diff
@@ -33,10 +33,18 @@ inline __wasi_size_t calculateBufferSize(const Container &Array) noexcept {
 template <typename T> struct WasiRawType {
   using Type = std::underlying_type_t<T>;
 };
-template <> struct WasiRawType<uint8_t> { using Type = uint8_t; };
-template <> struct WasiRawType<uint16_t> { using Type = uint16_t; };
-template <> struct WasiRawType<uint32_t> { using Type = uint32_t; };
-template <> struct WasiRawType<uint64_t> { using Type = uint64_t; };
+template <> struct WasiRawType<uint8_t> {
+  using Type = uint8_t;
+};
+template <> struct WasiRawType<uint16_t> {
+  using Type = uint16_t;
+};
+template <> struct WasiRawType<uint32_t> {
+  using Type = uint32_t;
+};
```

Comment:
It seems that this part doesn't pass the CI. But according to my own clang-format this may be formatted like this.

My clang-format version is 15.0.7.

Reply:
I will check this tmr.

Reply:
This will be fixed by #2299